src/goto-instrument/loop_utils.cpp

```diff
@@ -116,6 +116,7 @@ void get_modifies_lhs(
     const pointer_arithmetict ptr(to_dereference_expr(lhs).pointer());
     for(const auto &mod : local_may_alias.get(t, ptr.pointer))
     {
+      if(mod.id() == ID_unknown) continue;
```

Comment:
Please remove this. Unrelated to this PR.

Reply:
Yes, this's been removed in the latest version.